### PR TITLE
Fixes #276

### LINKED ISSUE #276: offline mode not working
Hi,
I am trying to generate a report using the offline mode.
The steps were the following:
- cloned the repo locally
- built with VS2015 ... success
- *clean* in order to delete the binaries
- build using the command line (build.cmd)
- run the ApiPort using the command line from the following folder:
   dotnet-apiport\bin\Release\ApiPort.Offline 

**This is the command line I used:**
`ApiPort analyze -f c:\temp\test.dll -t ".NET CORE, Version=5.0" -o test.html -r html`
**The output of the tool is the following:**
```
Detecting assembly references                     [Done]
Sending data to service                           [Failed]

There was an unknown error code from the service: 'ProxyAuthenticationRequired'.
  Please make sure you have the most up-to-date version.
```

Why is the tool still trying to send data to the service?
How can I use it totally offline?

Thanks

### PR DESCRIPTION
**Pull request by conniey**
Copy the unity.config file that sets up the use of Microsoft.Fx.Portability.Offline

### CODE REVIEW

File: build/postbuild.ps1 [hunk 1]
```diff
@@ -59,8 +59,6 @@ foreach($nuspec in $nuspecs)
 				$item.Pushed = $true;
 			}
 		}
-		
-		$item
```
Comment (twsouthwick) [inline build/postbuild.ps1]: I had this in there so you could see the status of whether it was pushed or not. Is there a reason you removed it?

Comment (conniey) [inline build/postbuild.ps1]: Ahh, all I saw was a truncated path like below. I'll add it back with Format-List.

```
Path
----
E:\git\repos\conniey\dotnet-apiport\bin\Release\Microsoft.Fx.Portability.MetadataReader\Microsoft.Fx.Portability.Met...
E:\git\repos\conniey\dotnet-apiport\bin\Release\Microsoft.Fx.Portability\Microsoft.Fx.Portability.1.4.0-alpha-00000....
E:\git\repos\conniey\dotnet-apiport\bin\Release\Microsoft.Fx.Portability.Offline\Microsoft.Fx.Portability.Offline.1....
E:\git\repos\conniey\dotnet-apiport\bin\Release\Microsoft.Fx.Portability.Reports.Html\Microsoft.Fx.Portability.Repor...
E:\git\repos\conniey\dotnet-apiport\bin\Release\Microsoft.Fx.Portability.Reports.Json\Microsoft.Fx.Portability.Repor...
```

Comment (twsouthwick) [inline build/postbuild.ps1]: Nice - what does the output look like now?

**Comment by twsouthwick** [comment]
LGTM

File: build/postbuild.ps1 (hunk 1 above)
Comment (conniey) [inline build/postbuild.ps1]:
```
Path    : E:\git\repos\conniey\dotnet-apiport\bin\Release\Microsoft.Fx.Portability.Reports.Html\Microsoft.Fx.Portabilit
          y.Reports.Html.2.0.0-alpha-00000.nupkg
Pushed  : False
Nuspec  : Microsoft.Fx.Portability.Reports.Html.nuspec
Package : Microsoft.Fx.Portability.Reports.Html.2.0.0-alpha-00000.nupkg

Path    : E:\git\repos\conniey\dotnet-apiport\bin\Release\Microsoft.Fx.Portability.Reports.Json\Microsoft.Fx.Portabilit
          y.Reports.Json.1.0.1-alpha-00000.nupkg
Pushed  : False
Nuspec  : Microsoft.Fx.Portability.Reports.Json.nuspec
Package : Microsoft.Fx.Portability.Reports.Json.1.0.1-alpha-00000.nupkg
```

**Pull request merged**